### DISCUSSION TITLE
Welcome to reactive Discussions!

### DISCUSSION THREAD

**Discussion by s-cork** [anvilistas/reactive#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bobincomox** [comment]
I have been an avid Anvil user for about 2 years now, and have been using anvil-labs atomic module for state. About a week ago two of my long running apps got errors regarding stack overflow. I realized from the start that the labs projects are not production ready but I used them anyways. Instead of spending time working out the problem with atomic, I switched over to reactive. It only took a few hours to get everything running again! I found it simpler than atomic to use, especially with the simple examples to go by. Thanks for making reactive available. I like that it has versions to lock into!

Bob St.John
